xbmc/utils/Geometry.h

```diff
@@ -98,13 +98,13 @@ template <typename T> class CPointGen
 };
 
 template<typename T>
-inline bool operator==(const CPointGen<T> &point1, const CPointGen<T> &point2) noexcept
+inline constexpr bool operator==(const CPointGen<T> &point1, const CPointGen<T> &point2) noexcept
```

Comment:
You can remove the `inline`

Reply:
True, but this is orthogonal to my change. The `inline` here was never necessary or useful.

Reply:
> True, but this is orthogonal to my change.

I disagree. `constexpr` implies `inline` in this case so `inline` can be removed (no matter whether it was necessary before).

Reply:
Oh, you're right. I thought `template` would also imply `inline`, but it doesn't - it's just that both turn the function into a weak symbol, and both suppress `-Wunused`.